src/App.php

```diff
@@ -168,6 +168,11 @@ public function processRequest(ServerRequestInterface $request, float $start_tim
 		);
 	}
 
+	public function processConsole(array $argv): void
```

Comment:
I really don't like this master App class tbh :)

What benefits do we get to use a method here, just to centralize it?

Console doesn't need the App class, it needs the dependencies, which are wrapped in the container. I don't see any benefit.. it feels wrong to use another class to initiate a class, which could be initialized directly.

As said, I'm really not a fan of it ;)